server/src/http-exception.filter.ts

```diff
@@ -1,6 +1,25 @@
 import { ExceptionFilter, Catch, ArgumentsHost, HttpException } from '@nestjs/common';
 import { Request, Response } from 'express';
 
+
+function unfoldedStackTrace(response, status) {
+  if (!response.response) {
+    return [{
+      code:   response.code,
+      title:  response.title,
+      detail: response.detail,
+      status,
```

Comment:
Maybe we could turn this into an object:

```
const errorObject = {
      code:   response.code,
      title:  response.title,
      detail: response.detail,
      status,
};
```

Just so that the shape of the error object is in one spot.

Reply:
way ahead of me! yep!